packages/mobile/src/utils/formatting.ts

```diff
@@ -37,13 +37,24 @@ export const getExchangeRateDisplayValue = (value: BigNumber.Value): string => {
   return new BigNumber(value).decimalPlaces(4).toFormat()
 }
 
-export const getFeeDisplayValue = (value: BigNumber.Value | null | undefined): string => {
-  return value
-    ? // Show 0.001 if fee > 0 and <= 0.001
-      BigNumber.max(value, new BigNumber(value).isZero() ? 0 : 0.001)
-        .decimalPlaces(4)
-        .toFormat()
-    : ''
+/**
+ * Rounds up to upper-bound fees for display
+ * Fails quietly (displays minimum value) on nonnegative inputs
+ * @param value fee amount, assumed nonnegative
+ * @param topLine true if display should be to two decimal places
+ */
+export const getFeeDisplayValue = (
+  value: BigNumber.Value | null | undefined,
+  topLine: boolean = false
+): string => {
+  if (!value || new BigNumber(value).isZero()) {
+    return ''
+  }
+  const decimals = topLine ? 2 : 3
+  const minDisplay = topLine ? 0.01 : 0.001
```

Comment:
These hardcoded values can be replaced easily with exponentiation on `decimals` (if we need other precisions than 2 and 3 places), e.g.,
`const minDisplay = 10 ** (0 - decimals)`
(we would also probably change `topLine` to `decimals: number = 2`)

Reply:
not really my area but 3 numbers after the decimal indicator can make it look like a thousands separator. Particularly because the symbols used for these are swapped in some locales, 

suggestion to do 2 or 4

Reply:
Thanks for the review! As per the linked issue, design requests that the dropdown figures are displayed to precisely 3 decimal places. I can ask the designers about this if you think it's an important distinction, though?

Reply:
i really do in fact i thought before we had made a decision to not to it this way.

Reply:
@isabellewei this was the previous conversation on number of decimal places